mypy/messages.py

```diff
@@ -1521,6 +1521,13 @@ def no_overridable_method(self, name: str, context: Context) -> None:
             context,
         )
 
+    def override_decorator_missing(self, name: str, context: Context) -> None:
+        self.fail(
+            f'Method "{name}" is not marked as override '
+            "but is overriding a method in a base class",
```

Comment:
I think saying it is "not marked as override" might be a bit confusing, perhaps this could be rephrased slightly like so?
```suggestion
            f'Method "{name}" is not marked by an override decorator'
            "but is overriding a method in a base class",
```

Reply:
I actually tried to copy the error message `pyright` uses. It's something like
```
Method "method1" is not marked as override but is overriding a method in class "Base"
```

_Note that pyright explicity mentions the base class here, whereas mypy does not. If that is useful, it could always be added later._